pkg/entity/db.go

```diff
@@ -30,10 +31,22 @@ var AutoMigrateTables = []interface{}{
 	FlagEntityType{},
 }
 
+func connectDB() (db *gorm.DB, err error) {
+	err = retry.Do(
+		func() error {
+			db, err = gorm.Open(config.Config.DBDriver, config.Config.DBConnectionStr)
+			return err
+		},
+		retry.Attempts(config.Config.DBConnectionRetryAttempts),
+		retry.Delay(config.Config.DBConnectionRetryDelay),
```

Comment:
btw, the default retryType is backoff.

https://github.com/avast/retry-go/blob/master/retry.go#L80-L87

Reply:
Awesome. By default 10 retries gives us 25.5 seconds of reconnect time. If we change the default attempts to 9, our retry time comes down to 12.7 seconds. That seems more like a happy medium between long enough and not too long to me. What do you think?